vi/news/_posts/2015-04-13-ruby-2-1-6-released.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: news_post
+title: "Phát hành Ruby 2.1.6"
+author: "usa"
+translator: "My Mai"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Phiên bản Ruby 2.1.6 đã được phát hành.
+
+Phiên bản phát hành lần này bao gồm việc sửa chữa bảo mật cho OpenSSL extension.
+Xem các chủ đề bên dưới để biết thêm chi tiết.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

Comment:
The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile: